sample-projects/sfm3019-arduino/README.md

```diff
@@ -0,0 +1,32 @@
+# SFM3019 Sample Code for Arduino
+
+This Arduino sample project shows the basic usage of Sensirion's SFM3019 flow
+sensor.
+
+## Compatibility
+
+The code herein is compatible with Sensirion's SFM3019 digital flow sensor.
+
+* Sensirion SFM3019 Sensor
+* Arduino (1.8)
+
+## Introduction
+
+Sensirion's developer page
+[developer.sensirion.com](https://developer.sensirion.com) provides more
+developer resources for different platforms and products.
+
+## Cloning this Repository
+
+```
+git clone --recursive https://github.com/Sensirion/embedded-sfm.git
+```
+
+## Building from the repository
+
+If you cloned the repository, prior to building the project you need to execute
```

Comment:
I would add a note to use the prepared release zip if unsure and broadly describe what the script does. About the script itself, can we call make prepare or is that not included in git bash?

Reply:
`make` is usually not available on Windows, that's why I generate the version from the script.